rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-getting-started-workflow.adoc

```diff
@@ -15,17 +15,21 @@ This document provides an overview of the ROSA with STS deployment workflow stag
 . xref:../rosa_getting_started_iam/rosa-installing-rosa.adoc#rosa-installing-rosa[Install the ROSA and OpenShift CLI tools and verify the AWS servce quotas]. Install and configure the ROSA CLI (`aws`) and the OpenShift CLI (`oc`). You can verify if the required AWS resource quotas are available by using the ROSA CLI.
-. xref:../rosa_getting_started_iam/rosa-creating-cluster.adoc#rosa-creating-cluster[Create a ROSA cluster] or xref:../rosa_getting_started_iam/rosa-aws-privatelink-creating-cluster.adoc#rosa-aws-privatelink-creating-cluster[Create a ROSA cluster using AWS PrivateLink]. Use the ROSA CLI (`rosa`) to create a cluster. You can optionally create a ROSA cluster with AWS PrivateLink.
-. xref:../rosa_getting_started_iam/rosa-accessing-cluster.adoc#rosa-accessing-cluster[Access a cluster]. You can configure an identity provider and grant cluster administrator privileges to the identity provider users as required. You can also access a newly-deployed cluster quickly by configuring a `cluster-admin` user.
+. xref:../rosa_getting_started_iam/rosa-creating-cluster.adoc#rosa-creating-cluster[Create a ROSA cluster] or xref:../../rosa_install_access_delete_clusters/rosa-aws-privatelink-creating-cluster.adoc#rosa-aws-privatelink-creating-cluster[Create a ROSA cluster using AWS PrivateLink]. Use the ROSA CLI (`rosa`) to create a cluster. You can optionally create a ROSA cluster with AWS PrivateLink.
+. xref:../rosa_getting_started_iam/rosa-accessing-cluster.adoc#rosa-accessing-cluster[Access a cluster]. You can configure an identity provider and grant cluster administrator privileges to the identity provider users as required. You can also access a newly deployed cluster quickly by configuring a `cluster-admin` user.
 . xref:../rosa_getting_started_iam/rosa-deleting-access-cluster.adoc#rosa-deleting-access-cluster[Revoke access to a ROSA cluster for a user]. You can revoke access to a ROSA cluster from a user by using the ROSA CLI or the web console.
 . xref:../rosa_getting_started_iam/rosa-deleting-cluster.adoc#rosa-deleting-cluster[Delete a ROSA cluster]. You can delete a ROSA cluster by using the ROSA CLI (`rosa`).
 
 [id="additional_resources_{context}"]
 [role="_additional-resources"]
 == Additional resources
 
-* For information about using the ROSA deployment workflow to create a cluster that uses the AWS Security Token Service (STS), see xref:../../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-sts-overview-of-the-deployment-workflow[Understanding the ROSA with STS deployment workflow].
+* For information about using the ROSA deployment workflow to create a cluster that uses the AWS Security Token Service (STS), see xref:../rosa-sts-getting-started-workflow.adoc#rosa-sts-overview-of-the-deployment-workflow[Understanding the ROSA with STS deployment workflow ].
```

Comment:
This file fell out of sync in the first phase of the restructure. It was my fault. Are the links still correct?